docs/src/appendix/notation.md

```diff
@@ -80,10 +80,11 @@ The following table also uses a few conventions that suffuse the source code and
 | ``\mathcal{L}^i(T)``                | `ℒⁱ`   | `ice_latent_heat(T, constants)`     | Temperature-dependent latent heat of deposition                                |
 | ``θ₀``                              | `θ₀`   | `RS.potential_temperature`          | (Constant) reference potential temperature for the anelastic formulation       |
 | ``p₀``                              | `p₀`   | `RS.surface_pressure`               | Base (surface) reference pressure                                              |
```

Comment:
is there a default for p₀? shall we mention here?

Reply:
the default is 101325 which is the mean sea level pressure on earth